src/test/java/fi/aalto/cs/apluscourses/TestHelper.java

```diff
@@ -17,7 +22,7 @@
 /**
  * A helper class to simplify testing plugin manipulation logics.
  */
-public abstract class PluginsTestHelper extends BasePlatformTestCase {
+public abstract class TestHelper extends BasePlatformTestCase {
```

Comment:
renamed and joined. there is now one aggregate test helper class that extends the `lightweight` test class of IJ, hopefully you don't mind :wink: